### ISSUE TITLE
Technodefirmacraft 2.3.1 crashes on startup.

### ISSUE THREAD

**Issue by aziel13** [Bunsan/TechNodeFirmaCraft#20]
Technodefirmacraft 2.3.1 crashes on startup.
http://pastebin.com/Bp6H1vcY

**Comment by AnodeCathode** [comment]
Well, as I just tested loading a new 2.3.1 server on both Windows and CentOS without issue, we're going to need more information in order to assist. 

Did you download the TechNodefirmacraft - 2.3.1.zip server package from the AT Website or install it with the ATLauncher client? If the client, what options did you select?

Did you customize the server at all before launching?

Guess I'll paste this blurb to reddit as well, as you seem to have logged the issue again there. Only one location is really necessary in the future.
What is the environment that the server is running in? I see that it's creeperhost hosted, but as I've not used them I have no idea what their hosting packages consist of.

Where exactly did this crash occur? Immediately on startup? Can we see the rest of the log?

Any and all detail you can provide will assist us in troubleshooting your issue.

**Comment by aziel13** [comment]
I downloaded the 2.3.1 from the atlauncher client. On the client I selected 360 days, fastcraft, aromabackup, streams, waila, prattle, and localized weather& storms with the default configuration settings provided.

This crash occurs immediately after startup. After creating a new instance to retest this issue without installing over my existing server I found that the issue is that the issue was that toomuchtime-1.7.10-2.4.0.tnfc-3-universal.jar and toomuchtime-1.7.10-2.4.0.tnfc-4-universal.jar were both included in the mod folder. Looking back at the folder I copied. It may be that it merged the mod folder instead of replacing it.
